### PR TITLE
small improvements

### PR DESCRIPTION
**Pull request by frankbu**

### CODE REVIEW

File: _docs/guides/bookinfo.md [hunk 1]
```diff
@@ -58,9 +63,7 @@ and outgoing calls for the services, providing the hooks needed to externally co
 via the Istio control plane, routing, telemetry collection, and policy enforcement
 for the application as a whole.
 
-Before you begin, if you haven't already done so, setup Istio by following the instructions
-corresponding to your platform [installation guide]({{home}}/docs/setup/) and then
-run the application using the instructions in one of the following sections.
+To start the application, follow the instructions, below, corresponding to your Istio runtime environment.
```
Comment (rshriram) [inline _docs/guides/bookinfo.md]: no commas after the "follow the instructions,.. below,...corresponding.." .. Basically the last two commas seem out of place, but take it with a grain of salt please :)..

File: _docs/guides/integrating-vms.md [hunk 2]
```diff
@@ -12,16 +12,16 @@ This sample deploys the Bookinfo services across Kubernetes and a set of
 Virtual Machines, and illustrates how to use Istio service mesh to control
 this infrastructure as a single mesh.
 
+> Note: this guide is still under development and only tested on GCE.
+
```
Comment (rshriram) [inline _docs/guides/integrating-vms.md]: GCE -> Google Cloud Platform.

Comment (rshriram) [reply _docs/guides/integrating-vms.md]: May be even add something like on IBM Bluemix or most other platforms where overlay network of Pods is isolated from VM network, VMs cannot initiate any direct communication to Kubernetes Pods when using Istio.

**Comment by ldemailly** [comment]
changes are in #562

**Pull request merged**